PIL/GifImagePlugin.py

```diff
@@ -454,14 +454,9 @@ def _get_local_header(fp, im, offset, flags):
     else:
         duration = 0
 
-    if "disposal" in im.encoderinfo:
-        disposal = int(im.encoderinfo['disposal'])
-        if not 0<= disposal <=3:
```

Comment:
This range check is not needed.
Because the higher bits is reserved, this option will not pollute another data.
And if it is higher than 0xffff(ushort), than will raise struct.error